src/libraries/Microsoft.Extensions.Primitives/tests/StringSegmentTest.cs

```diff
@@ -480,17 +482,6 @@ public void StringSegment_Equals_String_Valid(string candidate, StringComparison
             Assert.Equal(expectedResult, result);
         }
 
-        [Fact]
```

Comment:
It's no longer valid (see added lines above). `Equals(null)` now returns `false` instead of throwing.
Discussed here: https://github.com/dotnet/runtime/pull/57395#discussion_r688980555

Reply:
Sounds fine with me